sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeed.cs

```diff
@@ -107,6 +107,11 @@ internal class ChangeFeed
                 return BlobChangeFeedEventPage.Empty();
             }
 
+            if (pageSize > Constants.ChangeFeed.DefaultPageSize)
+            {
+                pageSize = Constants.ChangeFeed.DefaultPageSize;
```

Comment:
Why is this needed?

Reply:
We want to limit page size to 5000.  The rest API list operations return a max of 5000 items per page.

Reply:
Just to be clear, we are limitting it to 5000 just to follow the convention, not because it acctually has any effect in our change feed implementation, right?

Reply:
That's correct.